aline-eks/microservices/deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Idk why everything is commented out 
Could look into using a makefile to split the functionality into commands instead of uncommenting/commenting out code when you need it